homeassistant/components/neurio_energy/sensor.py

```diff
@@ -139,9 +149,19 @@ def __init__(self, data, name, sensor_type, update_call):
 
         if sensor_type == ACTIVE_TYPE:
             self._unit_of_measurement = POWER_WATT
-            self._attr_state_class = STATE_CLASS_MEASUREMENT
+            self._attr_device_class = DEVICE_CLASS_POWER
         elif sensor_type == DAILY_TYPE:
             self._unit_of_measurement = ENERGY_KILO_WATT_HOUR
+            self._attr_device_class = DEVICE_CLASS_ENERGY
+
+    @property
+    def last_reset(self) -> datetime | None:
```

Comment:
Note that we're simplifying calculating the last reset time in the recorder statistics so this may be able to be replaced by the new state class `STATE_CLASS_TOTAL_INCREASING` for the energy sensor. This was changed a couple of days ago and we still haven't merged the dev docs update.

Reply:
Yup, this has landed now.

@deftdawg Please git rebase this PR onto the latest `dev` and make the adjustments as per:
<https://developers.home-assistant.io/blog/2021/08/16/state_class_total/>